utils/cryptography/encryption.go

```diff
@@ -0,0 +1,65 @@
+package crytography
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"io"
+	"os"
+
+	"golang.org/x/crypto/nacl/secretbox"
+)
+
+// EncryptionKey for encrypting secrets
+var EncryptionKey, _ = base64.StdEncoding.DecodeString(os.Getenv("ENCRYPTION_KEY"))
+var byteEncryptionKey [32]byte
+
+// Both what the encryption key length should be
+var keyLength = 32
+
+var (
+	// ErrEncryptedFieldTooLarge - the sku was invalid
+	ErrEncryptedFieldTooLarge = errors.New("Encrypted field is greater than 16 KB - this must be chunked")
+)
+
+// Init copies the specified encryption key into memory once
+func Init() {
+	copy(byteEncryptionKey[:], []byte(EncryptionKey))
+}
+
+// EncryptMessage uses SecretBox to encrypt the message
+func EncryptMessage(field []byte) (encrypted []byte, nonceString [24]byte, err error) {
+	var nonce [24]byte
+
+	// The key argument should be 32 bytes long
+	if len(EncryptionKey) != keyLength {
```

Comment:
should we move this to the Init function if we don't expect `EncryptionKey` to change afterwards?

Reply:
Moved this to be passed in via variable 👍 

https://github.com/brave-intl/bat-go/pull/347/commits/d2677ac4f51f1a32192564125370e450ea346a45